command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
On Thu, Jul 28, 2016 at 03:15:48PM -0700, Mike Brown wrote:
> @wking said: `I don't think we want humans reading code to be part
> of certification, and am perfectly happy certifying a shim/core
> system as a single black box entity [1].  Folks interested in
> consuming the certified runtime should use the same shim/core system
> that was certfied. `
> 
> Heh, no offense but I don't trust black boxes ;-)

It's hard to have fair/objective certification if you relax the
black-box constraint on certification.

> My point was "if" someone wanted to certify a product with a
> different type of interface "then" we could certify that product
> against the runtime spec after inspecting the shim.  I'm not saying
> we "have" to I'm saying it's a viable point of discussion.

I'm not in the certification group, so this is not my call.  But I'd
be very skeptical of a certification body that says:

  runC v1.0.0 is compliant with runtime-spec v1.0.0 on Linux amd64
  because we've run ocitools on it with the runC-provided API 1↔2 shim
  v1.0.0 and Alice confirmed that API 1↔2 shim v1.0.0 isn't magic.

What would “Alice confirmed that API 1↔2 shim v1.0.0 isn't magic” look
like?  How do you challenge Alice if you don't agree?